Framework/Muon/src/ApplyMuonDetectorGrouping.cpp

```diff
@@ -210,17 +211,37 @@ void ApplyMuonDetectorGrouping::exec() {
   WorkspaceGroup_sptr muonWS = convertInputWStoWSGroup(inputWS);
   clipXRangeToWorkspace(*muonWS, options);
 
-  auto ws = createAnalysisWorkspace(inputWS, false, options);
-  auto wsRaw = createAnalysisWorkspace(inputWS, true, options);
-
   const std::string wsName = getNewWorkspaceName(options, groupedWSName);
   const std::string wsRawName = wsName + "_Raw";
+  std::vector<std::string> wsNames = {wsName, wsRawName};
+
+  const std::string wsunNormName = wsName + "_unNorm";
```

Comment:
we can define `"_unNorm` at the top of the file in an anonymous namespace https://github.com/mantidproject/mantid/pull/22653/files#diff-ef8bcda06d8d7c7d120651e19b290d73R68